src/community/smart-data-loader/src/test/java/org/geoserver/smartdataloader/postgis/PostGisPrefixAppSchemaVisitorTest.java

```diff
@@ -0,0 +1,16 @@
+package org.geoserver.smartdataloader.postgis;
```

Comment:
Copyright header is missing in new files (reporting here, please check them all)

Reply:
Right, added to the new added files.  I will create a new PR adding the missing headers for all the already existing files.